frontend/src/App.js

```diff
@@ -1,22 +1,21 @@
-import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
-import { useAuthContext } from "./hooks/useAuthContext";
-import Blog from "./pages/blog/Blog";
-import Navbar from "./components/Navbar";
-import CreateArticle from "./pages/blog/CreateArticle";
-import Article from "./pages/blog/Article";
-import Login from "./pages/Login";
-import Signup from "./pages/Signup";
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom'
+import { useAuthContext } from './hooks/useAuthContext'
+import Blog from './pages/blog/Blog'
+import Navbar from './components/Navbar'
+import CreateArticle from './pages/blog/CreateArticle'
+import Article from './pages/blog/Article'
+import Login from './pages/Login'
+import Signup from './pages/Signup'
 
 function App() {
-  const { user } = useAuthContext();
+  const { user } = useAuthContext()
   return (
     <div className="App">
       <BrowserRouter>
         <Navbar />
         <div className="pages">
           <Routes>
             <Route path="/" element={<Navigate to="/blog" />} />
-            <Route path="/blog" element={<Blog />} />
```

Comment:
This line deletion should be in a separated commit :wink: 
it's quite hard to find what is the "fix minor bug" in such a big formatting!

Reply:
is there something @atxr can do here?

Reply:
I think it's to late, let's not waste time there.
Thanks to the ci it won't happen anymore ☺️